Description: Fix a bug in case FG_SCENERY is not set.
Author: Rebecca Palmer 

--- a/src/wizard_funcs.cxx
+++ b/src/wizard_funcs.cxx
@@ -335,6 +335,7 @@
     else if (!def_fg_scenery.empty())
     {
         fg_scenery = def_fg_scenery;
+        prefs.set("fg_scenery", def_fg_scenery.c_str());
     }
     else if (fg_root_->size() > 0)
     {
